code/game/mecha/mecha.dm

```diff
@@ -469,15 +469,17 @@
 
 /obj/mecha/attack_hand(mob/living/user)
 	user.changeNext_move(CLICK_CD_MELEE)
-	user.do_attack_animation(src)
 	log_message("Attack by hand/paw. Attacker - [user].",1)
 
 	if((HULK in user.mutations) && !prob(deflect_chance))

```

Comment:
If/when this PR is done, `HULK` really probably should be abstracted to `attack_hulk`.